integration-libs/cdc/organization/administration/cdc-administration.module.ts

```diff
@@ -5,51 +5,15 @@
  */
 
 import { NgModule } from '@angular/core';
-
-import { CmsConfig, provideDefaultConfig } from '@spartacus/core';
-import {
-  unitsCmsConfig,
-  userCmsConfig,
-  UserListService,
-} from '@spartacus/organization/administration/components';
+import { UserListService } from '@spartacus/organization/administration/components';
 import { B2BUserService } from '@spartacus/organization/administration/core';
 import { CdcB2BUserService } from './cdc-b2b-user.service';
 import { CdcUserListService } from './cdc-user-list.service';
 
 @NgModule({
   providers: [
-    //to override UserListService in ListComponent
-    provideDefaultConfig(<CmsConfig>{
-      cmsComponents: {
-        ManageUsersListComponent: {
-          providers: [
-            {
-              provide: UserListService,
-              useExisting: CdcUserListService,
-            },
-            userCmsConfig.cmsComponents?.ManageUsersListComponent?.providers ||
-              [],
-          ],
-        },
-      },
-    }),
-    //to override B2BUserService in UserDetailsComponent, UnitUserListComponent
+    { provide: UserListService, useClass: CdcUserListService },
     { provide: B2BUserService, useClass: CdcB2BUserService },
-    //to override B2BUserService in UnitUserRolesCellComponent
-    provideDefaultConfig(<CmsConfig>{
-      cmsComponents: {
-        ManageUnitsListComponent: {
-          providers: [
-            {
-              provide: B2BUserService,
-              useExisting: CdcB2BUserService,
-            },
-            unitsCmsConfig.cmsComponents?.ManageUnitsListComponent?.providers ||
-              [],
-          ],
-        },
-      },
-    }),
```

Comment:
Do you mean this configuration is not needed, as I hoped?

Reply:
reverted this change